noir-projects/noir-contracts/contracts/test_contract/src/main.nr

```diff
@@ -405,7 +403,7 @@ contract Test {
     #[aztec(private)]
     fn set_constant(value: Field) {
         let mut note = TestNote::new(value);
-        storage.example_constant.initialize(&mut note, false, GrumpkinPoint::zero(), GrumpkinPoint::zero());
+        storage.example_constant.initialize(&mut note, GrumpkinPoint::zero(), GrumpkinPoint::zero());
```

Comment:
I think it would be nice to add a small comment here that we are not failing because the testnote is never broadcast. It is in the note itself, but this would usually cause a failure if actually broadcasting 🤷

Reply:
Agree. Added the comment in https://github.com/AztecProtocol/aztec-packages/pull/6741/commits/c998e604096d57f45861528c9bc3f8f4633715ee